dashboard/test/ui/features/foundations/hamburger.feature

```diff
@@ -1,13 +1,104 @@
-@no_mobile
 @single_session
 Feature: Hamburger dropdown
 
+  @eyes_mobile
+  Scenario: Signed-out user sees the hamburger on mobile
+    Given I am on "http://code.org/"
+    And I open my eyes to test "Hamburger Menu in English Signed-out"
+    And I rotate to portrait
+    And I wait for 0.5 seconds
+    Then I wait until element "#hamburger-icon" is visible
+    And I scroll the "#hamburger-icon" element into view
+    And I click selector "#hamburger-icon"
+    And I see no difference for "Hamburger Menu: English Signed-out"
+    And I close my eyes
+
+  @eyes_mobile
+  Scenario: Signed-out user sees Teach, About and Legal dropdowns in hamburger
+    Given I am on "http://code.org/"
+    And I open my eyes to test "Hamburger Dropdowns in English Signed-out"
+    And I rotate to portrait
+    And I wait for 0.5 seconds
+    Then I wait until element "#hamburger-icon" is visible
```

Comment:
Are the back to back wait statement necessary? I feel like I remember that they are, but just wanted to have this clarified here in the PR. Was it because the second wait statement waits for 2 seconds and it needs more time?

Reply:
the 0.5 second wait is to account for the rotation. Most other examples if rotating to portrait make use of the wait after rotating